src/pages/mesh/release/index.md

```diff
@@ -16,7 +16,23 @@ import UpdateNotice from '/src/_includes/update-notice.md'
 
 # Release notes
 
-The following sections list updates to API Mesh for Adobe Developer App Builder. Refer to the [Upgrade version](upgrade.md) for more information on upgrading.
+The following sections list updates to API Mesh for Adobe Developer App Builder.
+
+To use the latest enhancements, update your CLI to the latest version:
+
+```bash
+aio plugins:install @adobe/aio-cli-plugin-api-mesh
+```
+
+## May 05, 2025
+
+This release contains the following changes to API Mesh:
+
+### Enhancements
+
+- Added response the `cf-ray` and `x-request-id` response headers to assist with [tracking and debugging](../advanced/headers.md#response-tracking-and-debugging).
```

Comment:
The language here needs to be updated. The headers are passed along in the source requests to track requests between mesh and individual sources.